data_steward/utils/bq.py

```diff
@@ -0,0 +1,8 @@
+from google.cloud import bigquery
+
+client = bigquery.Client()
```

Comment:
This cannot be mocked.  This needs to exist in a function.  preferably something like:

```
def get_client(project_id=None):
    """
    Get a client for a specified project.
    """"
    if project_id:
        return bigquery.Client(project=project_id)

    raise RuntimeError("You must specify a project to create a client.")
```